client/src/App.tsx

```diff
@@ -61,6 +88,60 @@ function App() {
     changeOptionalTestStatuses();
   }, [runOptionalTests]);
 
+  /*
+   * Checks the anchor's TOML for NETWORK_PASSPHRASE and validates that
+   * its the correct string depending on runOnMainnet
+   */
+  const checkAnchorNetwork = useCallback(async () => {
+    let url = domain.includes("http") ? domain : "https://" + domain;
```

Comment:
Can we reuse the 'getValidDomain' function below so we don't duplicate this logic?